components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -210,10 +228,12 @@ export default function () {
                             <span className="font-medium text-gray-500 dark:text-gray-50 truncate" title={p.info.branch}>{p.info.branch}</span>
                         </div>
                         <span className="flex-grow" />
-                        <ItemFieldContextMenu menuEntries={prebuildContextMenu(p)} />
+                        {!props.isAdminDashboard && <ItemFieldContextMenu menuEntries={prebuildContextMenu(p)} />}
                     </ItemField>
                 </Item>)}
             </ItemsList>
+            {(!isLoadingPrebuilds && prebuilds.length === 0 && props.isAdminDashboard) &&
+                <div className="mt-3">No prebuilds triggered yet.</div>}
```

Comment:
**suggestion**: Since you added an empty state here, could we adjust the spacing here to provide some visual balance? You could borrow the proposal layout from https://github.com/gitpod-io/gitpod/issues/5022. 🏓

Reply:
Done!